.github/workflows/e2e-testing.yml

```
@@ -19,15 +19,17 @@ jobs:
 
       - name: Run Playwright tests
         run: make test-e2e
 
-      - uses: actions/upload-artifact@v3
+      - name: Upload Playwright report
+        uses: actions/upload-artifact@v4
         if: always()
         with:
           name: playwright-report
-          path: playwright-report/
+          path: | 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)